### DISCUSSION TITLE
set html attributes

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1961] [Q&A] [answered]
besides style props and classes ,Is there any  way to get or set the html attributes with nicegui ,like 'id','src'...?

**Comment by falkoschindler** [comment, marked as answer]
"HTML attributes" are nothing else than "props". So you can use `.props()` to set them.
But "id" is used internally by NiceGUI. There's an ongoing discussion about whether the user should be able to set it: https://github.com/zauberzeug/nicegui/discussions/1583.
